install/getting-started-knative-app.md

```diff
@@ -91,20 +91,28 @@ assigned an external IP address.
 1. To find the IP address for your service, enter:
 
    ```shell
-    kubectl get svc knative-ingressgateway --namespace istio-system
+   # In Knative 0.2.x or prior versions, we use `knative-ingressgateway`.
+   INGRESSGATEWAY=knative-ingressgateway
 
-    NAME                     TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                                      AGE
-    knative-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
+   # In Knative 0.3.x the use of `knative-ingressgateway` is deprecated.
+   # We should use `istio-ingressgateway` since `knative-ingressgateway`
+   # will be removed in 0.4.
+   if kubectl get configmap config-istio -n knative-serving &> /dev/null; then
+       INGRESSGATEWAY=istio-ingressgateway
+   fi
 
+   kubectl get svc $INGRESSGATEWAY --namespace istio-system
+
+   NAME                     TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                                      AGE
+   xxxxxxx-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
```

Comment:
```suggestion
   istio-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
```

Reply:
GLOBAL. Since you've explained that knative-ingressgateway is being deprecated and will eventually be removed, makes sense to demo using `istio-ingressgateway`